README.md

```diff
@@ -140,4 +140,36 @@ await httpClient.post('identity/resources/auth/v1/user', {
     // When providing vendor-host, it will replace(<...>) https://<api>.frontegg.com with vendor host
    'frontegg-vendor-host': 'acme.frontegg'
 });
-```
\ No newline at end of file
+```
+
+### Validating JWT manually
+If required you can implement your own middleware which will validate the Frontegg JWT using the `IdentityClient`
+
+First, let's import the `IdentityClient`
+```javascript
+const { IdentityClient } = require('@frontegg/client');
+```
+
+Then, initialize the client
+```javascript
+const identityClient = new IdentityClient({ FRONTEGG_CLIENT_ID: 'your-client-id', FRONTEGG_API_KEY: 'your-api-key' });
+identityClient.validateIdentityOnToken(token, { roles: ['admin'], permissions: ['read'] });
```

Comment:
why do we need this line? It is used later in line 165 (user = IdentityClient.validateIdentity....)

Reply:
haha I just wrote that